Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/wrapper/impl/ConceptEntryWrapperCreator.java

```diff
@@ -79,6 +91,13 @@
 
                 }
             }
+            
+            //adding reviewflag in wrapper
+            revReq = dbManager.getEntry(entry.getId(),entry.getWordnetId(),true);
```

Comment:
getting a comment for a concept, should only the concept id. there is no reason for any other parameters, isn't there?